scripts/form-utils.js

```diff
@@ -114,22 +103,18 @@ function postToEmailServer(params, successCallback, errorCallback) {
     xhr.timeout = 5000;
     xhr.ontimeout = errorCallback;
     xhr.responseType = 'json';
-
-    if (params.newsletters) {
-      xhr.setRequestHeader("Content-type", "application/x-www-form-urlencoded");
-      xhr.send(serialize(params));
```

Comment:
I set up the basket side more this way -- form urlencoded. The response is JSON though.